pkg/mutation/system.go

```diff
@@ -107,12 +108,20 @@ func (s *System) Mutate(obj *unstructured.Unstructured, ns *corev1.Namespace) (b
 			}
 			if cmp.Equal(original, obj) {
 				if *MutationLoggingEnabled {
-					logAppliedMutations("Oscillating mutation.", original, allAppliedMutations)
+					logAppliedMutations("Oscillating mutation.", original, allAppliedMutations, nil)
```

Comment:
The mutation ID should be generated at the beginning and be carried through all mutation logs to make it easier to track the behavior of a given request all the way through.

Reply:
Done